db/Models/news.models.js

```diff
@@ -94,25 +94,61 @@ const insertCommentByArticle = (article_id, body) => {
       }
 
       const commentValues = Object.values(body);
-      console.log(commentValues);
+      // console.log(commentValues);
 
       let queryString = `INSERT INTO comments (body, votes, author, article_id) VALUES (%L) RETURNING *`;
       const formatString = format(queryString, commentValues);
 
       return db.query(formatString).then((data) => {
-        console.log(data.rows);
+        // console.log(data.rows);
         return data.rows[0];
       });
     });
 };
 
+const updateVotes = (article_id, inc_votes) => {
+  if (typeof inc_votes !== "number") {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
```

Comment:
The manual typechecking isn't really necessary because if the wrong data type makes it to the pgsql query, it will send back an error for us. This means that we don't need to do this step - our psql error handler can catch the error and handle the response